src/pip/_internal/wheel.py

```diff
@@ -791,24 +791,33 @@ def should_use_ephemeral_cache(
     if req.constraint:
         # never build requirements that are merely constraints
         return None
+
     if req.is_wheel:
-        if not should_unpack:
+        if must_build:
             logger.info(
                 'Skipping %s, due to already being wheel.', req.name,
```

Comment:
It would be good to decide if this log message really needs to be specific to one of the two cases.

Reply:
yes, any opinion about that?

Reply:
Or remove the message completely? In my use cases (i.e. `pip wheels -r requirements.txt`), it's not really useful to know that some requirements were wheels in the first place.

Reply:
Perhaps this should be the topic of another PR?